guides/common/modules/con_shellhooks-plugin.adoc

```diff
@@ -1,25 +1,34 @@
 [id="shellhooks_{context}"]
 = Shellhooks
 
-With webhooks, one {Project} event can only be mapped to one API call.
+With webhooks, you can only map one {Project} event to one API call.
 For advanced integrations, where a single shell script can contain multiple commands, you can install a {SmartProxy} shellhooks plugin that exposes executables using a REST HTTP API.
 
-A webhook can then be configured to reach out to a {SmartProxy} API to run a predefined shellhook, which, for example, can contain commands or edit files.
+You can then configure a webhook to reach out to a {SmartProxy} API to run a predefined shellhook.
+The shellhook can for example contain commands or edit files.
 
-Scripts must be placed in `/var/lib/foreman-proxy/shellhooks` as executables with only alphanumeric characters and underscores in the name.
+You must place the scripts in `/var/lib/foreman-proxy/shellhooks` as executables with only alphanumeric characters and underscores in the name.
 
-The HTTPS payload is passed using standard input, optional command line arguments can be provided using X-Shellhook-Arg-1 _to_ N.
+You can pass input to shellhook script through the webhook payload.
+This input is redirected to standard input of the shellhook script.
+You can pass arguments to shellhook script using HTTP headers in format X-Shellhook-Arg-1 to X-Shellhook-Arg-99.
```

Comment:
IMHO needs some form of highlighting fot your example format.

Reply:
Good catch! Adding monospace.